doc/specs/homepage.md

```diff
@@ -12,9 +12,11 @@ The homepage is the initial landing page for all authenticated users. It provide
 
 ## Business rules.
 
-The homepage contains the following sections:
+The homepage is a collection of customizable sections. A section is a small block of content, limited in scope. The user may personalize their homepage by adding, removing, & re-ordering sections. Sections that would otherwise appear empty do not appear.
 
-* **Site News**: An announcement containing critical information about the site. This section must appear at the top of the page & may not be removed by the user.
+In addition, a "Site News" section appears at the top. It is used for critical announcmenets for hte site. If the "Site News" section contains content, then the section appears at the top of the page & cannot be removed. Otherwise, the section does not appear.
```

Comment:
By site news do you mean the module that the user can add/remove from their page or the notification? I think it should be up to the user if they want a "Site news" module or not. Instead unread site news should be presented in a modal window (or similar presentation) when the user signs in. If site news is published while the user is browsing the site then it should be presented as a notification.